src/Widgets/NavigationWidget/SubComponents/MetaNavigation.tsx

```diff
@@ -57,10 +57,13 @@ export const MetaNavigation = connect(function MetaNavigation({
 
 const PortalLink = connect(function PortalLink({
   root,
+  widget,
 }: {
   root: HomepageInstance
   widget: NavigationWidgetInstance
 }) {
+  if (widget.get('slimDesign')) return null
```

Comment:
* [ ] See other comment regarding container vs child logic and added `widget` prop.